src/oca_github_bot/tasks/check_cla.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Camptocamp 2018
+# Copyright (c) Akretion 2014
+from ..cla import CLAChecker
+from ..queue import getLogger, task
+
+_logger = getLogger(__name__)
+
+
+_logger = getLogger(__name__)
+
+
+@task()
+def check_cla(owner, repo, pull_user, pr_number, action):
```

Comment:
can you add a `@switchable` decorator (see examples in other tasks) so it can be disabled by config.